sql/2021/ecommerce/android_ios_app_links_ecomm_sites.sql

```diff
@@ -0,0 +1,32 @@
+#standardSQL
```

Comment:
Updated the spreadsheet with this query as well using the July dataset.

Reply:
Adding the sheet tabs as well: `Native APP Links`

Reply:
@bobbyshaw - @rockeynebhwani and I chatted offline. I think this query won't be much use this year as well. While the `well-known.js` makes it much better, we'll want to do response body parsing on `/.well-known/assetlinks.json` and `/.well-known/apple-app-site-association` because some sites have empty bodies. This needs some updated work on `well-known.js`

Example = https://www.allbirds.com/.well-known/assetlinks.json